<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cut_ch_model_dialog</class>
 <widget class="QDialog" name="cut_ch_model_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dsf cutModelTool by wangdonghao</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(81, 81, 81);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>222</height>
       </rect>
      </property>
      <attribute name="label">
       <string>1.准备</string>
      </attribute>
      <widget class="QPushButton" name="create_knives_pb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>创建刀片</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mirror_knives_pb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>对称刀片</string>
       </property>
      </widget>
      <widget class="QPushButton" name="open_knives_sheet_pb">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>20</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>口</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>222</height>
       </rect>
      </property>
      <attribute name="label">
       <string>2.切身体</string>
      </attribute>
      <widget class="QPushButton" name="boolean_doit_pb">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>切割身体</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>选中身体：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rename_body_pb">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>全部重命名为body_*_Low</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>选中所有身体切片：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>（将身体命名为body_Low）</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>222</height>
       </rect>
      </property>
      <attribute name="label">
       <string>3.切衣服</string>
      </attribute>
      <widget class="QPushButton" name="boolean_once_doit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>切割衣服</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>132</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>选择单个刀片和模型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>(先将衣服命名为body_Low)</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>222</height>
       </rect>
      </property>
      <attribute name="label">
       <string>4.重命名</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>选中模型：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rename_as_body_pb">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>单个重命名为body_*_Low</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rename_clothes_pb">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>全部重命名:clothes_*_Low</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>23</y>
         <width>132</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>选中骨骼+模型：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rename_as_cloth_pb">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>单个重命名为clothes_*_Low</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>222</height>
       </rect>
      </property>
      <attribute name="label">
       <string>5.约束</string>
      </attribute>
      <widget class="QPushButton" name="add_constraint_pb">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>建立约束</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>222</height>
       </rect>
      </property>
      <attribute name="label">
       <string>6.增加切换属性</string>
      </attribute>
      <widget class="QPushButton" name="add_attr_pb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="text">
        <string>增加切换属性</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>选中大圈：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="conn_editor_pb">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>180</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>连接编辑器&gt;&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
